cloudify_aws/rds/tests/test_instance.py

```diff
@@ -105,7 +108,7 @@ def test_create_raises_UnknownServiceError(self):
             instance.create(ctx=_ctx, resource_config=None, iface=None)
 
         self.assertEqual(
-            str(error.exception),
+            text_type(error.exception),
```

Comment:
Same question. No change is necessary, just curious.

Reply:
Same as above